BFit/Controller/LoginViewController.swift

```diff
@@ -39,7 +39,7 @@ class LoginViewController: UIViewController {
     @IBAction func login(_ sender: Any) {
         let def = UserDefaults.standard
         def.set(true, forKey: "is_loggedIn")
-        def.set(2, forKey: "id")
+        def.set(1, forKey: "id")
```

Comment:
We just have to leave a mental reminder for ourselves to remove this hardcode when the login endpoint comes back.